test/parser/parsing_table_test.cpp

```diff
@@ -158,6 +166,9 @@ TEST(parser, ParsingTable3) {
   table.SetProductions(grammar);
   table.BuildTable();
 
+  std::vector<std::string> tl;
+  ASSERT_EQ(table.GetErrors(), tl);
+
```

Comment:
for a single instance new variable is not required.
use `std::vector<std::string>{}`